ios/Podfile

```diff
@@ -23,6 +23,8 @@ target 'ZulipMobile' do
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
   use_flipper!
+  pod 'RNHourFormat', :path => '../node_modules/react-native-hour-format'
```

Comment:
Was this (and some similarly purposed lines in Android files) added as a result of running `react-native link`? Unfortunately this library's setup instructions don't mention it, but I'd expect additions like this to be made unnecessary by React Native's autolinking feature (there's a doc for that somewhere).

I'd suggest reverting all changes except the ones in `src/` and running `yarn add react-native-hour-format`, and don't run a `react-native link` command. I suspect that will be sufficient setup, assuming autolinking treats this library the same way it's been treating others. (I can't remember if that ends up making a small change to an Android file after running on Android or running `tools/test android --full`, but I definitely don't expect it to change the Podfile.)

Reply:
Sure I have restore these linking changes and also test it out every thing is working fine